guides/databases/phpmyadmin.md

```diff
@@ -0,0 +1,51 @@
+# Intro
+
+---
+
+>**PHPMyAdmin** is a free software tool written in PHP, intended to handle  the administration of MySQL over the Web. phpMyAdmin supports a wide range  of operations on MySQL and MariaDB. Frequently used operations (managing  databases, tables, columns, relations, indexes, users, permissions, etc) can  be performed via the user interface, while you still have the ability to  directly execute any SQL statement.
+
+### Features provided by the program include:
+
+
+- Web interface
+
+- MySQL and MariaDB database management
```

Comment:
How can I benefit from Maria DB database management? Explain whatever you state in a succinct way. 😄

Reply:
Mostly we can say it's just the same as MySQL, but it's kind of a forked version of MySQL with advanced features like: --it's better at handling both large and small scale data better than former. 
-MariaDB has had a significant boost in terms of security features like internal security and password check, PAM and LDAP authentication, Kerberos, user roles, and strong encryption over tablespaces, tables and logs.
-MySQL has no support for extensibility, but MariaDB is built to be extensible at every layer—client, cluster, kernel and storage.
And a lot others, https://www.google.com/amp/s/opensourceforu.com/2018/04/why-mariadb-scores-over-mysql/amp/